README.md

```diff
@@ -246,11 +249,19 @@ The bond-level features are concatenated with the bond feature vectors before th
 
 Similar to molecule-, and atom-level features, the bond-level features are scaled by default. This can be disabled with the option `--no_bond_features_scaling`.
 
+### Spectra
+
+One of the data types that can be trained with Chemprop is "spectra". Spectra training is different than other datatypes because it considers the predictions of all targets together. Targets for spectra should be provided as the values for the spectrum at a specific position in the spectrum. The loss function for spectra is SID, spectral information divergence. Alternatively, Wasserstein distance (earthmover's distance) can be used for both loss function and metric with input arguments `--metric wasserstein --spectra_loss_function wasserstein`.
```

Comment:
Good catch. I started just having this as a spectra-specific argument where you could use an alternative spectra loss function but then I changed my mind later, thinking that it might be nice to have the flexibility to institute other loss functions for other dataset types as well. Though not currently implemented for anything else. Will send out updated test scripts.